app/src/main/res/layout/activity_login.xml

```diff
@@ -135,31 +135,31 @@
         app:layout_constraintTop_toTopOf="@+id/guideline3" />
 
     <ImageButton
-        android:id="@+id/fbButton"
+        android:id="@+id/googleButton"
         android:layout_width="100dp"
         android:layout_height="100dp"
         android:layout_alignStart="@+id/upper_poly"
         android:layout_alignTop="@+id/upper_poly"
         android:layout_marginStart="60dp"
         android:layout_marginTop="10dp"
         android:background="@color/transparent"
-        android:src="@drawable/ic_facebook"
+        android:src="@drawable/login_google"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="@+id/guideline3" />
 
     <ImageButton
-        android:id="@+id/googleButton"
```

Comment:
Use underscores while naming in xml everywhere.

Reply:
Done!